todoman/model.py

```diff
@@ -702,7 +716,20 @@ def expire_lists(self, paths):
     def delete_list(self, name):
         self._conn.execute("DELETE FROM lists WHERE lists.name = ?", (name,))
 
-    def todo(self, id):
+    def todo(self, id, from_db=False):
```

Comment:
I'd rather call this flag `read_only`, because rather than expose *how* we read the `todo`, that naming reflects what the caller cares about (_"I want to edit this todo"_).

Or did I miss some scenarios where this wouldn't make sense?

What do you think?